packages/@react-spectrum/combobox/docs/ComboBox.mdx

```diff
@@ -0,0 +1,767 @@
+  <Item textValue="Add to queue">
+    <Add />
+    <Text>Add to queue</Text>
+    <Text slot="description">Add to current watch queue.</Text>
+  </Item>
+  <Item textValue="Add review">
+    <Draw />
+    <Text>Add review</Text>
+    <Text slot="description">Post a review for the episode.</Text>
+  </Item>
+    <Item textValue="Subscribe to series">
+    <Bell />
+    <Text>Subscribe to series</Text>
+    <Text slot="description">Add series to your subscription list and be notified when a new episode airs.</Text>
+  </Item>
+  <Item textValue="Report">
+    <Alert />
+    <Text>Report</Text>
+    <Text slot="description">Report an issue/violation.</Text>
+  </Item>
+</ComboBox>
+```
+
+## Props
+
+<PropTable component={docs.exports.ComboBox} links={docs.links} />
+
+## Visual options
+
+### Label alignment and position
+[View guidelines](https://spectrum.adobe.com/page/combo-box/#Label-position)
+
+By default, the label is positioned above the ComboBox. The `labelPosition` prop can be used to position the label to the side.
+The `labelAlign` prop can be used to align the label as "start" or "end". For left-to-right (LTR) languages, "start" refers to the left most edge of the ComboBox and "end" refers to the right most edge.
+For right-to-left (RTL) languages, this is flipped.
+
+```tsx example
+<ComboBox label="Favorite Animal" labelPosition="side" labelAlign="end">
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+
+### Quiet
+[View guidelines](https://spectrum.adobe.com/page/combo-box/#Quiet)
+
+```tsx example
+<ComboBox label="Favorite Animal" isQuiet>
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+
+### Disabled
+[View guidelines](https://spectrum.adobe.com/page/combo-box/#Disabled)
+
+```tsx example
+<ComboBox label="Favorite Animal" isDisabled>
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+
+### Read only
+[View guidelines](https://spectrum.adobe.com/page/combo-box/#Read-only)
+
+```tsx example
+<ComboBox label="Favorite Animal" isReadOnly selectedKey="red panda">
+  <Item key="red panda">Red Panda</Item>
+  <Item key="cat">Cat</Item>
+  <Item key="dog">Dog</Item>
+  <Item key="aardvark">Aardvark</Item>
+  <Item key="kangaroo">Kangaroo</Item>
+  <Item key="snake">Snake</Item>
+</ComboBox>
+```
+
+### Custom widths
+[View guidelines](https://spectrum.adobe.com/page/combo-box/#Width)
+
+```tsx example
+<ComboBox label="Favorite Animal" width="size-6000">
```

Comment:
Maybe add a maxWidth here as well like we have in picker to prevent the page needing to horizontally scroll on mobile?

```suggestion
<ComboBox label="Favorite Animal" width="size-6000" maxWidth="100%">
```